src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleExtensions.scala

```diff
@@ -414,6 +426,15 @@ class GoogleExtensions(val directoryDAO: DirectoryDAO, val accessPolicyDAO: Acce
 
         group = groupOption.getOrElse(throw new WorkbenchExceptionWithErrorReport(ErrorReport(StatusCodes.NotFound, s"$groupId not found")))
 
+        members <- group match {
+          case accessPolicy: AccessPolicy => if (isConstrainable(accessPolicy.id.resource, accessPolicy.actions)) {
```

Comment:
checking `accessPolicy.actions` alone is not enough, also need to check roles which can be though of as a collections of actions